app/src/main/java/dev/vendicated/vencord/MainActivity.java

```diff
@@ -49,7 +52,15 @@ protected void onCreate(Bundle savedInstanceState) {
             return;
         }
 
-        wv.loadUrl("https://discord.com/app");
+        Intent intent = getIntent();
+        if(Objects.equals(intent.getAction(), Intent.ACTION_VIEW)) {
```

Comment:
needs some room to breathe
```suggestion
        if (Objects.equals(intent.getAction(), Intent.ACTION_VIEW)) {
```
and pls gitignore the .idea folder

Reply:
Fixed